src/components/pokemon/pokemon.parser.ts

```diff
@@ -10,10 +11,68 @@ class PokemonParser implements Parser {
     }
 
     public Parse(): Pokemon[] {
-        return this.gameMaster
+        let rv = this.gameMaster
```

Comment:
I know it is easier to write shorthand, but for a better readability please write full variable names. I actually don't know what you mean with `rv`.